draft-ietf-jose-hpke-encrypt.md

```diff
@@ -300,6 +303,17 @@ this document, the valid combinations of the KEM, "kty" and "crv" are shown in
 {: #ciphersuite-kty-crv title="JWK Types and Curves for JOSE-HPKE Ciphersuites"}
 
 
+## Key Usage Guidelines for JOSE-HPKE
+
+To ensure predictable key usage within JOSE-HPKE, the following restrictions and guidelines are introduced:
+
+1. **New Key Use Values**
+   The following values are registered in the "JSON Web Key Use" registry to explicitly identify the roles of keys in HPKE operations:
+   - **HPKE-Sender:** A key intended for use in the sender role of HPKE operations, performing encryption and key encapsulation.
+   - **HPKE-Receiver:** A key intended for use in the receiver role of HPKE operations, performing decryption and key decapsulation.
```

Comment:
JWK has this:

"Because a core goal of this specification is for the resulting representations to be compact, it is RECOMMENDED that the name be short -- not to exceed 8 characters without a compelling reason to do so."

Maybe "HPKEauth" for sender role (since it is used for authentication) and "HPKE" for receiver role?

Reply:
Updated to "HPKEauth" and "HPKErecv"